docs/finalproject/week-1/lesson.md

```diff
@@ -51,27 +51,34 @@ Once you have your team and your assignment, start by sorting out the basic admi
 
 
 
-## Agile Development (60 minutes)
+## 2. Agile Development (60 minutes)
 
 
-### What is Agile Development? (30-45 minutes)
+### 2.1. What is Agile Development? (30-45 minutes)
+*Faciliated by Engineering Mentors*
 
-TO DO 
+Before we determine what we're going to build we should discuss how we're going to build it. The Engineering TAs will lead us through a discussion on Agile best practices using this deck - [What is Agile Development?](https://docs.google.com/presentation/d/1p9AV-pz_VpM0B9Lo6K6WUNANF1ycXgEPU4SbpgNzzSI/edit#slide=id.p) 
+
+### 2.2. What is a Product Manager? (25-30 minutes)
+*Faciliated by Product Managers*
 
-### What is a Product Manager? (25-30 minutes)
 To work in an Agile way it's best to be part of a cross-functional team, so we've asked some Product Managers (PM) to join us for the final project. To introduce who they are and what their role is your PM will run through this [What is a Product Manager?](https://docs.google.com/presentation/d/1W8H8V0Fxor-XPcCY7kuXPCWbA9xTqS1vW1RMVCcCZRc/edit#slide=id.p) presentation. 
 
   *A note for PMs* - don't forget to create your own copy of the presentation and update the 'Who am I?' section.
 
-### Project Management Boards (5 minutes)
-Project Management boards are a tool that you can use to help you organise how you work.
+### 2.3. Project Management Boards (5 minutes)
+*Led by students* 
+
+Project Management boards are a tool that you can use to help you organise how you work. 
 
 ```txt
-Create a [project board](https://help.github.com/en/articles/about-project-boards) in the repo for your tasks and stories
+Create a copy of this Project Board to manage your work - [Project Board](https://trello.com/b/wkQd3R5r/team-name-backlog)
```

Comment:
Links won't work in code blocks (fences with backticks) - you can't use markdown syntax there

Reply:
Happy to change that - should I open another PR?

Reply:
I'd add it to this one, it's all part of the same thing